apis/Google.Cloud.Bigtable.Admin.V2/Google.Cloud.Bigtable.Admin.V2.Tests/BigtableTableAdminClientTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax.Grpc;
+using Google.Protobuf;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.Admin.V2.Tests
+{
+    public class BigtableTableAdminClientTest
+    {
+        [Fact]
+        public async Task DropAllRows_Valid_Request()
+        {
+            var tableName = new TableName("project", "instance", "table");
+            await DropAllRows_ValidateArguments<NotImplementedException>(tableName);
+        }
+
+        [Fact]
+        public async Task DropAllRowse_Validate_TableName()
```

Comment:
Typo: Rowse
(I know you're really just checking that I actually review the tests ;)

Reply:
You caught me :)